package.json

```diff
@@ -70,6 +70,7 @@
     "eslint": "^6.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-eslint-comments": "^3.1.1",
+    "eslint-plugin-flowtype": "^5.1.3",
```

Comment:
Had to add this to commit. It errs out otherwise 😢

Reply:
that's ok. i'll figure this out later